.npmignore

```diff
@@ -0,0 +1,12 @@
+.github/
+.vscode/
+coverage/
+example/
+src/
+babel.config.js
+global.d.ts
+jest.config.js
+tsconfig.json
+
+**/__tests__/
+**/__mocks__/
```

Comment:
A safer approach to using `.npmignore` is to configure a "files" attribute in your package. This will ensure only the files described there + README, package and license will be exported with your package. You can always test how the package will look like by running `npm pack`

Reply:
I tried that before, but for some weird reason, the result of ts compilation included the `src` files inside `dist`.
I updated it here and worked well.